src/openrct2/ride/Ride.cpp

```diff
@@ -5399,7 +5399,7 @@ bool ride_has_adjacent_station(Ride* ride)
 
 bool ride_has_station_shelter(Ride* ride)
 {
-    auto stationObj = ride->GetStationObject();
+    const auto stationObj = ride->GetStationObject();
```

Comment:
```suggestion
    const auto* stationObj = ride->GetStationObject();
```
I've probably not seen that in the last PR, now is a good opportunity :)